CHANGELOG.md

```diff
@@ -50,6 +50,7 @@ Internal:
 - Add tests for panel component (PR [#500](https://github.com/alphagov/govuk-frontend/pull/500))
 - Add tests for skip-link component (PR [#498](https://github.com/alphagov/govuk-frontend/pull/498))
 - Add tests for file-upload component (PR [#504](https://github.com/alphagov/govuk-frontend/pull/504))
+- Add tests for select component (PR [#506](https://github.com/alphagov/govuk-frontend/pull/506))
```

Comment:
Can we document the 'bug fixes' (select relying on input component) separately as changes?

Reply:
Yep, that sounds good. Added.

Reply:
👍